plugins/out_es/es.c

```diff
@@ -36,6 +36,7 @@
 #include "es.h"
 #include "es_conf.h"
 #include "es_bulk.h"
+#include "fluent-bit/flb_macros.h"
```

Comment:
It seems this is not necessary (and when we do, we use #include <fluent-bit/flb_macros.h> )

Reply:
oops overzealous editor inclusion. fixed and force pushed to keep commit message correct.